app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -76,17 +76,28 @@ def parse_physical_server(node)
         :model                  => node.model,
         :serial_number          => node.serialNumber,
         :field_replaceable_unit => node.FRU,
-        :computer_system        => {
-          :hardware             => {
-            :networks  => [],
-            :firmwares => [] # Filled in later conditionally on what's available
-          }
-        }
+        # Filled in later conditionally on what's available
+        :computer_system        => {:hardware => {:networks => [], :firmwares => []}}
       }
       new_result[:computer_system][:hardware] = get_hardwares(node)
+      save_host_relationship(node)
       return node.uuid, new_result
     end
 
+    def save_host_relationship(node)
```

Comment:
Hey @rodneyhbrown7 , I figured out this process(which use the core to save the objects) before, it isn't the problem. The problem is that the relationship is saved in the Host object, so to use the `ems_refresh/save_inventory_physical_infra.rb` is unfeasible considering the structure proposed by RH guys.

The only way that we can use this, is converting the Host object found in a Hash, adding it as a "sub hash" in the Physical server hash and setting up a child key to host in `save_physical_server` method in `ems_refresh/save_inventory_physical_infra.rb` file.

But it looks like a workaround.